lighthouse-core/test/results/sample_v2.json

```diff
@@ -4546,23 +4546,28 @@
       "helpText": "Run the [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/) and the [Structured Data Linter](http://linter.structured-data.org/) to validate structured data. [Learn more](https://developers.google.com/search/docs/guides/mark-up-content)."
     }
   },
-  "runtimeConfig": {
-    "environment": [
-      {
-        "name": "Device Emulation",
-        "description": "Nexus 5X"
-      },
-      {
-        "name": "Network Throttling",
-        "description": "562.5 ms HTTP RTT, 1474.5600000000002 Kbps down, 675 Kbps up (DevTools)"
-      },
-      {
-        "name": "CPU Throttling",
-        "description": "4x slowdown (DevTools)"
-      }
-    ],
-    "blockedUrlPatterns": [],
-    "extraHeaders": {}
+  "runtimeSettings": {
```

Comment:
wdyt `configSettings`? that's what we call them elsewhere.

Reply:
sg done